docs/content/deployments/admin/deploying-on-vercel.md

```diff
@@ -0,0 +1,197 @@
+---
+description: 'Learn step-by-step.'
+addHowToData: true
+---
+
+# Deploy Admin to Vercel
+
+In this document, you’ll learn how to deploy the admin dashboard to Vercel.
+
+## Prerequisites
+
+### Medusa Components
+
+You must have a [Medusa backend](../../development/backend/install.mdx) installed along with an [admin dashboard plugin](../../admin/quickstart.mdx).
+
+### Required Accounts
+
+- [Vercel Account](https://vercel.com/)
+- [GitHub Account](https://github.com/): Only required if you’re deploying through the Vercel website.
+
+:::note
+
+If you want to use another Git Provider, it’s possible to follow along with this guide, but you’ll have to perform the equivalent steps in your Git Provider.
+
+:::
+
+### Required Tools
+
+- [Git CLI](../../development/backend/prepare-environment.mdx#git): Only required if you’re deploying through the Vercel website.
+
+---
+
+## Step 1: Create GitHub Repository
+
+:::note
+
+This step is only required if you’re deploying from the Vercel website. However, it’s highly recommended to connect your Vercel project to a Git repository for a better developer experience.
+
+:::
+
+Before you can deploy your admin dashboard, you need to create a GitHub repository and push the code base to it. To do that:
+
+1. On GitHub, click the plus icon at the top right, then click New Repository.
+2. You’ll then be redirected to a new page with a form. In the form, enter the Repository Name.
+3. Scroll down and click Create repository.
+
+### Push Code to GitHub Repository
+
+The next step is to push the code to the GitHub repository you just created.
+
+After creating the repository, you’ll be redirected to the repository’s page. On that page, you should see a URL that you can copy to connect your repository to a local directory.
+
+Copy the link. Then, open your terminal in the directory that holds your Medusa backend codebase and run the following commands:
+
+```bash
+git init
+git remote add origin <GITHUB_URL>
+```
+
+Where `<GITHUB_URL>` is the URL you just copied.
+
+Then, add, commit, and push the changes into the repository:
+
+```bash
+git add .
+git commit -m "initial commit"
+git push
+```
+
+After pushing the changes, you can find the files in your GitHub repository.
+
+---
+
+## Step 2: Configure Build Script
+
+In the `package.json` of the Medusa backend, add or change a build script for the admin:
+
+```json
+"scripts": {
+  // other scripts
+  "build:admin": "medusa-admin build --deployment",
+}
+```
+
+### Additional Build Options
+
+Aside from `--deployment`, you can use the following options when building your admin for deployment:
+
+- `--backend` or `-b`: a string specifying the URL of the Medusa backend. The default here is the value of the environment variable `MEDUSA_BACKEND_URL`. For example, `medusa-admin build --deployment --backend example.com`
```

Comment:
Add note: If the users use this param, they can not set the MEDUSA_BACKEND_URL in their Vercel dashboard.